examples/trials/pix2pix-pytorch/pix2pix.py

```diff
@@ -0,0 +1,149 @@
+import os
+import logging
+import time
+import argparse
+from collections import namedtuple
+import numpy as np
+import torch
+import nni
+from nni.utils import merge_parameter
```

Comment:
If this and `test.py` are the only two files that involve NNI related changes. We should throw away the rest of the files, and ask users to clone the original repo. We can import files from the original repo by modifying import path.

Reply:
That should be doable. I will fix that.